File: fibonacci.java
import java.util.Scanner;

public class Fibonacci {
  public static void main(String[] args) {
    Scanner scanner = new Scanner(System.in);
    
    System.out.println("Digite um número: ");
    int n = scanner.nextInt();
    
    if (n == 0 || n == 1) {
      System.out.println("O número " + n + " pertence à sequência de Fibonacci.");
      return;
    }
    
    int a = 0, b = 1, c;
    for (int i = 2; i <= n; ++i) {
      c = a + b;
      a = b;
      b = c;
    }
    
    if (c == n) {
      System.out.println("O número " + n + " pertence à sequência de Fibonacci.");
    } else {
      System.out.println("O número " + n + " não pertence à sequência de Fibonacci.");
    }
  }
}

File: Fibonacci.py
def fibonacci(n):
  if n == 0 or n == 1:
    return True
  a, b = 0, 1
  for i in range(2, n + 1):
    a, b = b, a + b
  return b == n

n = int(input("Digite um número: "))

if fibonacci(n):
  print("O número", n, "pertence à sequência de Fibonacci.")
else:
  print("O número", n, "não pertence à sequência de Fibonacci.")
  